styles.css

```diff
@@ -466,3 +467,528 @@ form .btn {
+  }
+
+  .nav-items > li {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 87px;
+    height: 40px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .framex > a {
+    width: 98px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .lang a {
+    text-decoration: none;
+    color: #000;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    margin-top: 10px;
+    padding-top: 5px;
+  }
+
+  .nav-items > li > a {
+    text-decoration: none;
+    color: #344563;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    width: 63px;
+    height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+  }
+
+  .header-section {
+    width: 100%;
+    height: 846px;
+    top: 71px;
+    background-color: #fff;
+    border-bottom-left-radius: 50px;
+  }
+
+  .content-section {
+    background-image: url(images/headerbg.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-bottom: 50px;
+    width: 1397.4px;
+    height: 721.46px;
+    left: 34.6px;
+  }
+
+  .content-section > .content3 {
+    gap: 12px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 1px;
+    position: absolute;
+    width: 920px;
+    height: 268px;
+    left: 260px;
+    top: calc(50% - 268px / 2);
+  }
+
+  .content3 > .content-text {
+    width: 920px;
+    height: 122px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .content3 > .content-text-p {
+    width: 920px;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .social-connect {
+    width: 117px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .snap {
+    display: none;
+  }
+
+  .snap1 {
+    width: 544px;
+    height: 448px;
+    padding: 24px 1px 24px 24px;
+  }
+
+  .snappy3 {
+    width: 544px;
+    height: 448px;
+    padding: 24px 24px 24px 1px;
+  }
+
+  .works {
+    background-color: #e5e5e5;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12px;
+    width: 100%;
+    border-radius: 16px;
+    outline: 4px solid yellow;
```

Comment:
- [ ] As you were working on the concerns from the previous review, you added this yellow outline which is not part of the design, please refactor it out(remove it).